config/autoload/album.global.php

```diff
@@ -6,7 +6,7 @@
                 Album\Action\AlbumListFactory::class,
 
             Album\Model\Repository\AlbumRepository::class =>
-                Album\Model\Repository\AlbumRepositoryFactory::class,
+                Album\Model\Repository\ZendDbAlbumRepositoryFactory::class,
```

Comment:
can we move the repository outside of Model ? So it will be `Album\Repository` instead of `Album\Model\Repository` . Thoughts ?